cli/flox/src/commands/init/node.rs

```diff
@@ -80,26 +84,45 @@ enum PackageJSONVersion {
     Unavailable,
     /// package.json specifies a version,
     /// and we found a search result satisfying that version constraint
-    Found(Box<SearchResult>),
+    Found(PackageInfo),
 }
 
+/// The information necessary to install a node ecosystem package for the init hook
+#[derive(Debug, Clone, PartialEq, Default)]
+struct PackageInfo {
+    pub version: Option<String>,
+    pub pkg_path: Vec<String>,
+}
```

Comment:
Oh its a "new" PackageInfo..
We already have a `PackageInfo` struct for listing packages that got me confused

Reply:
`pkg_path` in manifest descriptors is also a String now while here its (still) a list. Maybe worth to look into for catalog polishing though

Reply:
there's a similar struct `ProvidedPackage` in the `init/mod.rs` complete with `impl From<SearchResult>`.